Michaelw/multiple indexes (#7005)
DataDog/documentation: content/en/logs/indexes/_index.md

  text: "Logging without Limits*"
---

Log Indexes provide fine-grained control over your Log Management budget by allowing you to segment data into value groups for

> differing retention, quotas, usage monitoring & billing. Indexes are located on the [Configuration page][1] in the Indexes section. Double click on them or click on the *edit* button to see more information about the number of logs that were indexed in the past 3 days, as well as the retention period for those logs:

Reviewer: ```suggestion
Log Indexes provide fine-grained control over your Log Management budget by allowing you to segment data into value groups for differing retention, quotas, usage monitoring, and billing. Indexes are located on the [Configuration page][1] in the Indexes section. Double click on them or click on the *edit* button to see more information about the number of logs that were indexed in the past 3 days, as well as the retention period for those logs:
```

{{< img src="logs/indexes/sample_trace_id.png" alt="enable index filters"  style="width:80%;">}}

## Update log retention

> The index retention setting determines how long logs will be stored and searchable in the index. You can set the retention to any value allowed in your account configuration.

Reviewer: ```suggestion
The index retention setting determines how long logs are stored and searchable in Datadog. You can set the retention to any value allowed in your account configuration.
```
